stan/math/prim/prob/multi_normal_lpdf.hpp

```diff
@@ -102,6 +103,24 @@ return_type_t<T_y, T_loc, T_covar> multi_normal_lpdf(const T_y& y,
   return lp;
 }
 
+template <bool propto, typename T_y, typename T_loc, typename T_covar,
+          require_all_not_vector_t<T_y, T_loc>* = nullptr>
+return_type_t<T_y, T_loc, T_covar> multi_normal_lpdf(const T_y& y,
+                                                     const T_loc& mu,
+                                                     const T_covar& Sigma) {
+  using lp_type = return_type_t<T_y, T_loc, T_covar>;
+
+  const size_t N = y.rows();
+  lp_type lp(0.0);
+
+  for (size_t n = 0; n < N; ++n) {
+    auto current_y = y.row(n);
```

Comment:
Is this code safe? What happens if `y` and `mu` have different lengths? Won't this just read off the end?

I'd like to see this fixed before it gets into the code base. Perhaps use a `check_matching_sizes` or something similar?